frontend/web/src/app/components/physical-print/physical-print-edit/physical-print-edit.component.html

```diff
@@ -22,23 +22,61 @@ <h2 *ngIf="creating">Printing a Digital Print</h2>
             <div *ngIf="creating" [ngClass]="{'has-error': !requiredFieldsForm.get('digitalPrintID').valid}">
               <select class="form-control" formControlName="digitalPrintID" title="Choose Digital Print"
                       [ngClass]="{'has-error': !requiredFieldsForm.get('digitalPrintID').valid}">
-                <option *ngFor="let _digitalPrint of this.digitalPrints; let i = index" [value]="_digitalPrint.id"
-                        title="{{_digitalPrint.id}}">
-                  {{_digitalPrint.id}}
+                <option *ngFor="let _digitalPrint of this.digitalPrints; let i = index" [value]="_digitalPrint.id">
+                  {{_digitalPrint.name}}
                 </option>
               </select>
             </div>
             <br>
             <div *ngIf="!creating">
               <input disabled value="{{physicalPrint.digitalPrintID}}" class="form-control">
             </div>
+            <div *ngIf="digitalPrint" class="alert alert-info" role="alert">
+              A Physical Part will be created for each Digital Part in the Digital Print, Please link them to the corresponding order
+            </div>
+
+            <div *ngIf="digitalPrint" class="container">
+
+              <div *ngFor="let dpart of this.getDigitalParts(this.digitalPrint); let i = index"  class="row">
+
+                <div class="col-md-2">
+                  <label>ID:</label> Generated
+                </div>
+                <div class="col-md-2">
+                  <label>Digital Part: </label> {{dpart.id}}
+                </div>
+                <div class="col-md-2">
+                  <label>Magics Label: </label> {{dpart.mkey}}
+                </div>
+
+
+                <div class="col-md-5">
+                  <select name="formOrderedPartID" class="selectpicker form-control" title="Ordered Part ID" #o (change)="this.physicalPartOrderArray[i] = o.value">
+                    <option style="display:none" value="">Select an Ordered Part ID</option>
+                    <option title="{{_orderedPart.id}}"
+                            *ngFor="let _orderedPart of this.orderedParts; let i = index"
+                            [value]=_orderedPart.id>
+                      {{_orderedPart.id}}
+                  </select>
+                </div>
+              </div>
+              <br>
+            </div>
+
+            <!--&lt;!&ndash;<app-digital-print-details-panel *ngIf="digitalPrint" [digitalPrint]="digitalPrint"-->
+                                             <!--[digitalParts]="digitalParts" [close]="false">-->
+            <!--</app-digital-print-details-panel>&ndash;&gt;-->
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)